modules/tinymce/CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The dialog `redial` API will now only rerender the changed components instead of the whole dialog #TINY-8334
 - The dialog API `setData` method now uses a deep merge algorithm to support partial nested objects #TINY-8333
 - The dialog spec `initialData` type is now `Partial<T>` to match the underlying implementation details #TINY-8334
+- Improved support for placing the caret before/after noneditable elements within the editor #TINY-8169
```

Comment:
```suggestion
- Improved support for placing the caret before or after noneditable elements within the editor #TINY-8169
```
Docs has said we can't use `/` as it's not always clear, so just replacing it with something else.

Reply:
Will try to remember that. But makes sense since it could mean `and` or `or` depending on how you read it.